bower.json

```diff
@@ -27,33 +27,5 @@
     "bootstrap": "~3.3.4",
     "angular": "~1.3.16"
   },
-  "ignore": [
```

Comment:
Should we be removing all of these or just the ones that are no longer relevant, eg. emailable-report.html?